testcafe/tests/drive/file_sharing_scenario.js

```diff
@@ -107,13 +108,34 @@ test(`[Mobile] Drive : Access a file public link, download the file, and check t
   })
   await t.navigateTo(data.sharingLink)
   await publicViewerPage.waitForLoading()
-
   await publicDrivePage.checkActionMenuPublicMobile('file')
+
+  //Download
+  await isExistingAndVisibile(selectors.btnMoreMenu, '[...] Button')
```

Comment:
Because now I check twice the button is here : once before clicking it for download, and once before clicking it for 'create cozy', so it seems to make more sense to get it out of the method.

I was thinking maybe I could add 2 methods like `clickOnDownloadButton` (https://github.com/cozy/cozy-drive/blob/testcafe/createcozy/testcafe/tests/drive/file_sharing_scenario.js#L113-L123) and `clickOnCreateCozy` (https://github.com/cozy/cozy-drive/blob/testcafe/createcozy/testcafe/tests/drive/file_sharing_scenario.js#L131-L139) instead of just putting it in the test.

Reply:
The thing is `checkActionMenuPublicMobile ` doesn't really do what it says now. So either it needs to be renamed, or the checks should move back in it.
But I don't really see why the fact that we added another check justifies extracting the code from it..?
adding another check on the menu changes the scope of the function though…

Reply:
I can leave this check in the function, but while we were talking about this issue on mattermost, you made me realize it was not very clear to check and open the menu in this method, so I changed it.

(and now `checkActionMenuPublicMobile` is `checkDesktopElementsNotShowingOnMobile`)